book/go/api/api.md

```diff
@@ -165,15 +165,15 @@ It is important to ensure that data returned is always immutable or unique to av
 
 ### Key
 
-Partition keys are `uint64` values.
+Partition keys are `byte array` values.
```

Comment:
I'd write "`[]byte`" or "byte slice" here. Arrays and slices in Go are different things.